vector/src/main/java/im/vector/app/features/home/room/detail/timeline/format/DisplayableEventFormatter.kt

```diff
@@ -214,11 +205,6 @@ class DisplayableEventFormatter @Inject constructor(
             EventType.STICKER                   -> {
                 stringProvider.getString(R.string.send_a_sticker)
             }
-            EventType.REACTION                  -> {
```

Comment:
This is the `formatThreadSummary()` pair of the above deletion, I deleted it for consistency. But let me know if this is a bad idea.